From 319e7cd590ae16460cfecb70715bf41ceec81d1e Mon Sep 17 00:00:00 2001
From: Jim Meyering <meyering@redhat.com>
Date: Tue, 9 Oct 2012 15:40:33 +0200
Subject: [PATCH 36/89] build: do not rely on automake's AM_TESTS_ENVIRONMENT

* tests/Makefile.am (TESTS_ENVIRONMENT): Rename from
AM_TESTS_ENVIRONMENT, since it is not honored in automake-1.11.3
after all.  This reverts commit v3.1-14-g2bd66d7.  For now, I'll
leave the following commit that made bootstrap.conf require 1.11.2.
Prompted by a report of test failure from Phillip Susi.
---
 tests/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 80d5525..cdc1c4b 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -98,7 +98,7 @@ sep = $(PATH_SEPARATOR)
 # variables that can perturb tests are unset or set to expected values.
 # The rest are envvar settings that propagate build-related Makefile
 # variables to test scripts.
-AM_TESTS_ENVIRONMENT =				\
+TESTS_ENVIRONMENT =				\
   tmp__=$$TMPDIR; test -d "$$tmp__" || tmp__=.;	\
   TMPDIR=$$tmp__; export TMPDIR;		\
   export					\
-- 
1.8.5.3

